OCC.Service.Webhost.Tests/Session/GetSessionTests.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System.Linq;
+using System.Linq.Expressions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OCC.Data;
+using OCC.Service.Webhost.Tests.Helpers;
+
+namespace OCC.Service.Webhost.Tests.Session
+{
+    [TestClass]
+    public class GetSessionTests
+    {
+        private Data.Session _expectedSession;
+        private Event _expectedEvent;
+        private Person _expectedSpeaker;
+
+        private Services.Session _actualSession;
+
+        [TestInitialize]
```

Comment:
I would make this a single test instead of making each assert statement its own test. Seems like you are testing a Event Session with one speaker.